docker-stack.yml

```diff
@@ -0,0 +1,74 @@
+version: '3.3'
+
+networks:
+  net:
```

Comment:
For consistency with the compose file, please rename the network to `elk`.

Reply:
Are you sure?  If you deploy the stack as "elk" you will then end up with a network named "elk_elk".

Reply:
Sorry for the late update @hastarin 

Yes, it's all right, please rename it to `elk`.